cli/tests/integration/mod.rs

```diff
@@ -97,3 +97,23 @@ async fn bootstrap_cli_test(
     set_env(tmp_path_str, &url);
     Ok((url, devices, org_id))
 }
+
+#[macro_export]
+macro_rules! assert_cmd_success {
+    (with_password=$pass:expr, $($cmd:expr),+) => {
+        $crate::assert_cmd_success!(__internal__ $($cmd),+ , "--password-stdin")
+            .write_stdin(format!("{password}\n", password = $pass))
+            .assert()
+            .success()
+    };
+    (__internal__ $($cmd:expr),+) => {
```

Comment:
What does the `__internal__` mean ?

Reply:
I use it to define the internal part of the macro, look how it's shared between the `with_password` and the other case

Reply:
Thanks !